pkg/ring/ring.go

```diff
@@ -75,26 +75,38 @@ type ReadRing interface {
 	HasInstance(instanceID string) bool
 }
 
-// Operation can be Read or Write
-type Operation int
-
-// Values for Operation
-const (
-	Read Operation = iota
-	Write
-	Reporting // Special value for inquiring about health
-
-	// BlocksSync is the operation run by the store-gateway to sync blocks.
-	BlocksSync
-
-	// BlocksRead is the operation run by the querier to query blocks via the store-gateway.
-	BlocksRead
-
-	// Ruler is the operation used for distributing rule groups between rulers.
-	Ruler
+// Operation describes which instances can be included in the replica set, based on their state.
+type Operation interface {
+	// ShouldExtendReplicaSet returns true if given a state of instance that's going to be
```

Comment:
I don't think that's correct.

Reply:
Fair enough. I think the current comment is hard to read because of the comma at `added to the replica set, the replica set size`. I thought adding the `,` suggested above could fix it.